integration_tests.py

```diff
@@ -71,6 +71,15 @@
 }
 LOG_PATTERN = re.compile(r"^[\d\-]{10} [\d:]{8} UTC [^\s]+ ([A-Z]+):")
 
+# Note for MacOSX
+# You can run
+# ```bash
+# OSX=true ./integration_tests.py ...
+# ```
+# To support docker-compose
+IS_OSX = os.getenv("OSX")
+DOCKER_COMPOSE_CMD = ["docker-compose"] if IS_OSX else ["docker", "compose"]
```

Comment:
I am sure there are DIRAC developers with MaxOSX, and they did not have to do this gymnastic. Maybe there's a better way of finding out what is the command to use here.

Reply:
This is more about using alternative containerisation technologies e.g. orbstack/colima/...

How about:

```suggestion
DOCKER_COMPOSE_CMD = shlex.split(os.environ.get("DOCKER_COMPOSE_CMD", "docker compose"))
```